Gaijinizer patch/CommonEvents_275.txt

```diff
@@ -4,31 +4,35 @@
 
 大人しく、私と逝け……
 > CONTEXT: CommonEvents_275/13/Dialogue/enemy4
-You are already over...It's mediocre, it makes
-me ...#MTLed
+You are already finished......
+
+Be quiet, and die with me......#Strate
```

Comment:
大人しく is a funny one to translate because it doesn't neatly fit into everyday English. It's literally her saying "Be a good boy" or "Do as you're told", feel free to localise it to something with that kind of feeling.